cicecore/cicedyn/dynamics/ice_dyn_evp.F90

```diff
@@ -1807,6 +1811,12 @@ subroutine stressC_T  (nx_block,    ny_block  , &
                     + shearU(i-1,j  )**2 * uarea(i-1,j  )) &
                     / (uarea(i,j)+uarea(i,j-1)+uarea(i-1,j-1)+uarea(i-1,j))
 
+         shearT    = (shearU(i  ,j  )    * uarea(i  ,j  )  &
+                    + shearU(i  ,j-1)    * uarea(i  ,j-1)  &
+                    + shearU(i-1,j-1)    * uarea(i-1,j-1)  &
+                    + shearU(i-1,j  )    * uarea(i-1,j  )) &
+                    / (uarea(i,j)+uarea(i,j-1)+uarea(i-1,j-1)+uarea(i-1,j))
```

Comment:
This isn't terribly important, but to reduce the number of flops and potentially also paging associated with strides, I'd save the denominator in a temporary scalar and reuse it.

Reply:
The compiler may do this for us, but I'll update it just to make sure.